docs/manual/installation/move-contao.de.md

```diff
@@ -77,18 +77,28 @@ Gib dein Passwort ein, wenn du danach gefragt wirst.
 {{< /tabs >}}
 
 
+## Einstellungen notieren
+Deine Einstellungen unter `System / Einstellungen` oder im Seitenroot notierst du dir am besten und trägst sie später erneut ein.
+
+{{% notice tip %}}
+Wenn du deine [Einstellungen in einer Umgebungsvariablen ablegst](https://docs.contao.org/manual/de/system/einstellungen/#umgebungsvariablen), entfällt dieser Schritt.
+{{% /notice %}} 
+
 ## Dateien übertragen
 Die folgenden Dateien und Ordner müssen vom Quell- zum Zielserver übertragen werden:
 
 - `files`
 - `templates`
 - `composer.json`
-- `composer.lock`
```

Comment:
This article is about moving _an existing_ Contao instance _as is_. This means you _must_ copy the `composer.lock`, otherwise you will not get the same dependencies as before.

Reply:
I only copied the composer.json and everything was fine. The dependicies are in the .json, aren't they?

Reply:
With the `composer.json` you define the dependencies that you want - in which version range. The `composer.lock` however stores what dependencies are _currently_ installed. If you run `composer install` without a `composer.lock`, then Composer will automatically execute a `composer update` which might result in a different dependency set than you had before. Typically when moving or deploying a Contao instance that is what you do _not_ want. Thus you need to copy the `composer.lock` too.

Reply:
OK, on the other hand: If composer.json is different to the .lock, then with the next update dependencies might be changed anyway.

Reply:
Not sure what you mean there. If you did not change anything in your `composer.json` before copying the Contao instance to another server, then the `composer.lock` hash will still match the `composer.json`.

Reply:
I agree to add the composer.lock